bigtable/use-cases/fraudDetection/terraform/scripts/send_transactions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Without associated tests, I worry about the maintenance burden of this script. Is there a way you can ensure this works as expected?